### ISSUE TITLE
[MarkupLM] Code for pre-training

### ISSUE THREAD

**Issue by Misterion777** [microsoft/unilm#918]
Hi, is it possible to have access to original training code of MarkupLM (CommonCrawl preprocess, tags masking, etc.) ?

**Comment by yash0307** [comment]
Hi, did you come across the code?

**Comment by Misterion777** [comment]
> Hi, did you come across the code?

Hi, no, I think its proprietary
